charts/zipkin/ci/extraManifests-values.yaml

```diff
@@ -0,0 +1,13 @@
+extraManifests:
```

Comment:
I was looking at our test, and I guess there's no way to test this, huh.. https://github.com/openzipkin/zipkin-helm/blob/master/charts/zipkin/templates/tests/test-connection.yaml